src/index.js

```diff
@@ -50,6 +51,27 @@ export class BlockSuggestor {
     this.generateBlockData = this.generateBlockData.bind(this);
   }
 
+  setModel(model) {
+    this.model = model;
+  }
+
+  getSuggestedBlocks = function () {
+    let inputSource = '';
```

Comment:
Use `description` for this variable name since it's not necessarily the input source but rather the text that it contains.